config/1.0.0/sample_config_1.0.0.json

```diff
@@ -188,126 +184,129 @@
     {
       "Applicability": {
         "Mission": "ICESat2",
-        "ShortNamePath": "ATL0[3-9]|ATL1[023]"
+        "ShortNamePath": "ATL0[3-9]|ATL1[023]",
+        "Variable_Pattern": "/$"
       },
-      "Applicability_Group": [
+      "Attributes": [
         {
-          "Applicability": {
-            "Variable_Pattern": "/$"
-          },
-          "Attributes": [
-            {
-              "Name": "Data_Organization",
-              "Value": "h5_trajectory"
-            }
-          ]
-        },
+          "Name": "Data_Organization",
+          "Value": "h5_trajectory"
+        }
+      ]
+    },
+    {
+      "Applicability": {
+        "Mission": "ICESat2",
+        "ShortNamePath": "ATL0[3-9]|ATL1[023]",
+        "Variable_Pattern": "/gt[123][lr]/geolocation/.*"
+      },
+      "Attributes": [
         {
-          "Applicability": {
-            "Variable_Pattern": "/gt[123][lr]/geolocation/.*"
-          },
-          "Attributes": [
-            {
-              "Name": "ancillary_variables",
-              "Value": "podppd_flag"
-            }
-          ]
-        },
+          "Name": "ancillary_variables",
+          "Value": "podppd_flag"
+        }
+      ]
+    },
+    {
+      "Applicability": {
+        "Mission": "ICESat2",
+        "ShortNamePath": "ATL03",
+        "Variable_Pattern": "/gt[123][lr]/geophys_corr/.*"
```

Comment:
Actually you added it because you wanted to? An Applicability doesn't need a Mission

Reply:
First up - I think this comment probably sums up why this change is a good thing to do 😆 

> So before it was picking up "Mission" from "Applicability" and then overriding the "ShortNamePath"?
This is me trying to understand where Mission came from.

Yeah, in this instance I was just migrating the content that was in the `Applicability` at the root level into each individual `Applicability_Group` that is now it's own item.

> An Applicability doesn't need a Mission

True. I had to check, but at a minimum only one of `Mission`, `ShortNamePath` or `Variable_Pattern` are required. Now I've typed that out, perhaps it makes more sense if `Variable_Pattern` can't be on it's own, so at a minimum `Mission` or `ShortNamePath` are required? An applicability with only `Variable_Pattern` would apply to _everything_. That seems like a bad plan.

Reply:
I thought about this some more... The reason that the schema has the option to define applicability solely by a variable pattern, I _think_, is exactly because of this old nesting. Before this change, the short name and/or mission could be in the applicability at the root of an item, and then the specific applicability in the `Applicability_Group` only needed a `Variable_Pattern` because it inherited the short name and mission from the root level. I will update the schema to remove the option of only a `Variable_Pattern` in the applicability. Thanks @flamingbear for making me think about this some more!

Reply:
...and here we go: [b9d232](https://github.com/nasa/earthdata-varinfo/pull/33/commits/b9d232a630f966eb7db107bb42b556dcd92092b8)